src/Casts/FirstPaymentActionsCast.php

```diff
@@ -1,10 +1,7 @@
 <?php
-
-
 namespace Laravel\Cashier\Casts;
 
 use Illuminate\Contracts\Database\Eloquent\CastsAttributes;
-use Illuminate\Support\Collection;
 
 class FirstPaymentActionsCast implements CastsAttributes
```

Comment:
If we stick with a plain json encode/decode, I think there's a native Eloquent cast available for that?

Reply:
I think the native `array` cast could work here. We then can drop this `FirstPaymentActionsCast`.

https://laravel.com/docs/8.x/eloquent-mutators#array-and-json-casting